src/glang/parser/parser.c3

```diff
@@ -1437,8 +1434,5 @@ function main : (argc: int, argv: u8[&][&]) -> int = {
         json_output:write_json_value(&program);
         json_output:close();
     }
-
-    // 4) Cleanup
-    file_contents:deallocate();
```

Comment:
Still need to call `unmap`, right?

Reply:
Added! Which is kinda funny when we're leaking literal the entire allocator

Reply:
We'll get there :)